operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -42,15 +47,36 @@ class TurbineLongTermGrossEnergy(object):
     """
 
     @logged_method_call
-    def __init__(self, plant):
+    def __init__(self, plant, UQ = 'Y', num_sim = 100, uncertainty_scada=0.005,
+                  uncertainty_wind_bin_thresh=(1, 3), uncertainty_max_power_filter=(0.8, 0.9), 
+                  uncertainty_correction_threshold=(0.85, 0.95)):
+    
         """
         Initialize turbine long-term gross energy analysis with data and parameters.
         Args:
          plant(:obj:`PlantData object`): PlantData object from which TurbineLongTermGrossEnergy should draw data.
-         max_power_filter(float): 
+         UQ:(:obj:`str`): choice whether to perform ('Y') or not ('N') uncertainty quantification
```

Comment:
Similarly, changing this to a boolean seems simpler for specifying UQ.

Reply:
Changed!